PubSub/src/PubSubClient.php

```diff
@@ -91,11 +97,17 @@ class PubSubClient
 
     const FULL_CONTROL_SCOPE = 'https://www.googleapis.com/auth/pubsub';
 
+    const GAPIC_KEYS = [
```

Comment:
These are here only to be declarative. Only use for this is in the `getGapicsFromConfig` method. If we want to deprecate the user option to override the GAPIC client that is used, we don't need this or the `getGapicsFromConfig` method.